test/example/cpp-unit.cpp

```diff
@@ -637,6 +637,19 @@ void test_effects() {
     assert_equal(effects.readsMutableStruct, false);
     assert_equal(effects.writesStruct, false);
   }
+
+  {
+    EffectAnalyzer effects(options, module);
```

Comment:
Do we have an effect analyzer gtest suite we could put this in instead?

Reply:
I looked for that first, but we don't.

Separately it would be nice to migrate these tests, I agree.